src/components/SupportUs/SendEmail/SendEmail.js

```diff
@@ -135,21 +154,39 @@ function SendEmail({ language }: Props) {
       from_name: name,
       reply_to: email,
       send_copy: sendCopy,
+      recaptchaKey,
     })
   }
 
-  const handleSubmit = (event: SyntheticEvent<HTMLButtonElement>) => {
-    event.preventDefault()
+  const handleSubmit = (recaptchaKey: string) => {
     if (name !== '' && email !== '' && phone !== '' && message !== '') {
-      const mailParams = createMail()
-      // TODO: Send email
-      // eslint-disable-next-line no-alert
-      alert(`Thanks for your interest! \nCheck your Tec email \n${mailParams.message}`)
-      setName('')
-      setEmail('')
-      setPhone('')
-      setMessage('')
-      setCopy(false)
+      const mailParams = createMail(recaptchaKey)
+      setIsLoading(true)
+      sendContactUsEmail(mailParams).then((result) => {
+        if (result.status === 1) {
```

Comment:
I think the code returned should be 200 (OK)

Reply:
@Josecisneros001 I mentioned the same in a previous revision, can you change it?